export const failwith = (str: string): void => {
  throw new Error(str);
};
